jdaviz/configs/imviz/helper.py

```diff
@@ -223,35 +224,40 @@ def load_data(self, data, data_label=None, show_in_viewer=True, **kwargs):
                     if (has_wcs and linked_by_wcs) or not linked_by_wcs:
                         self.app.add_data_to_viewer(show_in_viewer, applied_label, visible=visible)
 
-    def link_data(self, link_type='pixels', wcs_fallback_scheme=None, wcs_use_affine=True):
+    def link_data(self, align_by='pixels', wcs_fallback_scheme=None, fast_approximation=True):
         """(Re)link loaded data in Imviz with the desired link type.
         All existing links will be replaced.
 
         Parameters
         ----------
-        link_type : {'pixels', 'wcs'}
+        align_by : {'pixels', 'wcs'}
             Choose to link by pixels or WCS.
 
         wcs_fallback_scheme : {None, 'pixels'}
             If WCS linking failed, choose to fall back to linking by pixels or not at all.
-            This is only used when ``link_type='wcs'``.
+            This is only used when ``align_by='wcs'``.
             Choosing `None` may result in some Imviz functionality not working properly.
 
-        wcs_use_affine : bool
+        fast_approximation : bool
             Use an affine transform to represent the offset between images if possible
             (requires that the approximation is accurate to within 1 pixel with the
             full WCS transformations). If approximation fails, it will automatically
-            fall back to full WCS transformation. This is only used when ``link_type='wcs'``.
+            fall back to full WCS transformation. This is only used when ``align_by='wcs'``.
             Affine approximation is much more performant at the cost of accuracy.
 
         """
-        from jdaviz.configs.imviz.plugins.orientation.orientation import link_type_msg_to_trait
+        from jdaviz.configs.imviz.plugins.orientation.orientation import align_by_msg_to_trait
         plg = self.plugins["Orientation"]
         plg._obj.wcs_use_fallback = wcs_fallback_scheme == 'pixels'
-        plg.wcs_use_affine = wcs_use_affine
-        plg.link_type = link_type_msg_to_trait[link_type]
+        plg.fast_approximation = fast_approximation
+        plg.align_by = align_by_msg_to_trait[align_by]
 
     def get_link_type(self, data_label_1, data_label_2):
+        logging.warning("DeprecationWarning: get_link_type has been replaced by "
```

Comment:
Why `logging.warning` instead of normal `warnings.warn`? Or you can also use the `@deprecated` decorator from `astropy.utils`?

This comment applies to all other similar occurrences.

Reply:
You know, I was just being lazy and copied what the first `DeprecationWarning` I `grep`ed in the repo was doing. I'll update it to use the astropy decorator.

Reply:
Updated!

Reply:
Are you sure it is updated? I still see `logging.warning` in the diff instead of `@deprecated`.

Reply:
Yes, I see the astropy deprecation in the diff. Refresh?

Reply:
I refreshed, but no change. You sure you pushed that commit out?

<img width="572" alt="Screenshot 2024-08-02 153813" src="https://github.com/user-attachments/assets/fa1861ee-2ea5-4800-9b5a-4302045edb6e">

Reply:
Ahh I wasn't paying attention to where this comment was, thought you were talking about the other file and forgot there was one here. One sec...